src/main/java/org/dave/compactmachines3/miniaturization/MultiblockRecipe.java

```diff
@@ -159,7 +160,7 @@ public void setPositionMap(String[][][] map) {
                         continue;
                     }
 
-                    referenceCount.put(content, referenceCount.getOrDefault(content, 0)+1);
+                    referenceCount.merge(content, 1, Integer::sum);
```

Comment:
Assuming this is an automated update from Java 8?

Reply:
Not automated, no. I manually modified this once I saw it was not using the proper solution since I was in that file for some other reason and it caught my eye.